doc(app deploy): Add info on how to deploy a living application (#1366)
bonitasoft/bonita-doc: md/get-bonita-provided-applications.md

> # Deploying the Bonita User and Admin Applications

Reviewer: ```suggestion
# Deploying the Bonita User and Administrator Applications
```

> # Deploying the Bonita User and Administrator Applications

Reviewer 2: ```suggestion
# Creating a Living Application based on an existing application
```

> This page describes how to get the two applications and how to deploy them into a bundle or the cloud.

Reviewer: ```suggestion
This page describes how to get the two applications and how to deploy them into a Tomcat bundle or the cloud.
```

Reviewer 2: Capital "C" with the Cloud :]

> This page describes how to get the two applications and how to deploy them into a Tomcat bundle or the cloud.

Reviewer 2: ```suggestion
This page describes the possibility not to create a Living Application from scratch but instead based on one of the two Applications provided by Bonita: the Bonita User Application and the Bonita Administrator Application.```

> ## How to get the applications 

Reviewer 2: ```suggestion
## Bonita applications
```

> 

Reviewer 2: ```suggestion
Starting with Bonita 2021.1, Bonita comes with two applications: Bonita User Application, and Bonita Administrator Applications, that you can customize to fit the needs of your end-users and administrators.
To know more about how to get them and deploy them, go to the dedicated pages for [Bonita User Application](user-application-overview.md) and Bonita Administrator Application](admin-application-overview.md).

## Application examples from the Community
You can also find examples of applications as projects contributed by members of our Community.   
To do so, visit https://community.bonitasoft.com/ and search in the "Contribute/Projects" section.
In the _Category_ filter, use the value "Application".
```

> You can get the applications in two different ways, either from the Studio welcome page or from the two repositories.

Reviewer: ```suggestion
You can get the applications in two different ways, either from the Studio welcome page or from their two GitHub repositories.
```

> You can get the applications in two different ways, either from the Studio welcome page or from their two GitHub repositories.

Reviewer 2: I would remove all the following content. I've added what was missing (deploying without BCD) in the overview pages of the 2 Bonita applications.

Moreover, the two repositories have two ways to get the applications.

To get the pages from the Studio welcome page, all you need to do is to click on the link for the corresponding application.

![Get application from Studio](images/application-deploy/studio-get-application.JPG)

This will open an import window, which will let you import the application.

> For a more technical user, there is a second way to get the applications. You can get it from it's corresponding GitHub repository. 

Reviewer: ```suggestion
For more technical users, there is a second way to get the application. You can get it from its corresponding GitHub repository. 
```

For the user application, you can find it in the [user application repository](https://github.com/bonitasoft/bonita-user-application/).
And for the admin one, you can find it in the [admin application repository](https://github.com/bonitasoft/bonita-admin-application/).
There are two ways to get the .bos file. You can either get it from the *Releases* on the right or by cloning the repository of your choice and building it using maven.
Building the repository will create a .bos file in the ```target``` folder. You can import this one into the development environment.
<br>Using this method is also useful if you want to get an older version of an application or if you need a weekly version that contains a bugfix.

Now that you got the applications, you can deploy them into your locally running server using the Studio deployment.

> A way to deploy the applications into a bundle or into your cloud platform is explained in the [applications](applications.md) page.

Reviewer: ```suggestion
A way to deploy the applications into a Tomcat bundle or into your cloud platform is explained in the [applications](applications.md) page.
```
